manifests/04-deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: pod-checkpointer-operator
       containers:
         - name: pod-checkpointer-operator
-          image: quay.io/ryan_phillips/pod-checkpointer-operator:v0.0.2
+          image: openshift/pod-checkpointer-operator:latest
```

Comment:
why are you not using v4.0 tag instead of latest?

Reply:
This will be replaced by the release render.  If you are using the raw manifest file to deploy, you are likely doing dev and want the latest image right? And if you use the v4.0 tag, you'll have to open a PR to updated it on each release.

Reply:
plus, it matches this https://github.com/openshift/pod-checkpointer-operator/blob/master/Makefile#L17

Reply:
> This will be replaced by the release render. 

Only if the tag is v4.0. With scheduler operator, when the tag was latest, it was not being replaced.

> If you are using the raw manifest file to deploy, you are likely doing dev and want the latest image right?

Agree that is fine for development. But for development, if you dev image is not in openshift org but in your personal org, you would have to update image path in manifest anyways.

Reply:
> Only if the tag is v4.0. With scheduler operator, when the tag was latest, it was not being replaced.

The image only need match this to be replaced https://github.com/openshift/pod-checkpointer-operator/blob/master/manifests/image-references#L8

Reply:
> The image only need match this to be replaced https://github.com/openshift/pod-checkpointer-operator/blob/master/manifests/image-references#L8

its fine then if thats the case.